### ISSUE TITLE
Improvements to ephys workflows guis

### ISSUE THREAD

**Issue by Dario55** [SainsburyWellcomeCentre/aeon_experiments#550]
Since many time we are operating the ephys workflows from remote, it would be useful to have a visual feedback of when we press shift f3 and shift f4 to either operate the commutator or turn the motor on and off. 
for the motor, it would be useful to know if the motor is on or off even if one is not in the room.

**Comment by NeuroThom** [comment]
Agreed, during development this has been a consistent "nice to have". In fact I thin this is important to have in the long run
